### DISCUSSION TITLE
How to do latex math in markdown

### DISCUSSION THREAD

**Discussion by UlisseMini** [Textualize/textual#1938] [Q&A] [answered]
How can I render markdown like `$\int_0^\pi f(x)dx = F(\pi) - F(0)$` as $\int_0^\pi f(x)dx = F(\pi) - F(0)$ in textual? Is this possible to do with some hacks, or very hard and I should consider a different approach? Are you considering adding this as a feature in future?

**Comment by willmcgugan** [comment, marked as answer]
Looks like there is a [Markdown extension](https://markdown-it-py.readthedocs.io/en/latest/plugins.html) for that. So we could add it to the Markdown widget.

**Reply by phineas-pta** [reply]
did it get added by any chance ?

**Reply by ninjamar** [reply]
Any chance there is an update on this issue?
